doc/Dashboard.md

```diff
@@ -0,0 +1,159 @@
+
+```js
+admin.dashboard(nga.dashboard()
+    .addCollection(nga.collection(post)
+        .name('recent_posts')
+        .title('Recent posts')
+        .perPage(5) // limit the panel to the 5 latest posts
+        .fields([
+            nga.field('published_at', 'date').label('Published').format('MMM d'),
+            nga.field('title').isDetailLink(true).map(truncate),
+            nga.field('views', 'number')
+        ])
+        .sortField('published_at')
+        .sortDir('DESC')
+        .order(1)
+    )
+    .addCollection(nga.collection(post)
+        .name('popular_posts')
+        .title('Popular posts')
+        .perPage(5) // limit the panel to the 5 latest posts
+        .fields([
+            nga.field('published_at', 'date').label('Published').format('MMM d'),
+            nga.field('title').isDetailLink(true).map(truncate),
+            nga.field('views', 'number')
+        ])
+        .sortField('views')
+        .sortDir('DESC')
+        .order(3)
+    )
+    .addCollection(nga.collection(comment)
+        .title('Last comments')
+        .perPage(10)
+        .fields([
+            nga.field('created_at', 'date')
+                .label('Posted'),
+            nga.field('body', 'wysiwyg')
+                .label('Comment')
+                .stripTags(true)
+                .map(truncate)
+                .isDetailLink(true),
+            nga.field('post_id', 'reference')
+                .label('Post')
+                .map(truncate)
+                .targetEntity(post)
+                .targetField(nga.field('title').map(truncate))
+        ])
+        .sortField('created_at')
+        .sortDir('DESC')
+        .order(2)
+    )
+    .addCollection(nga.collection(tag)
+        .title('Tags publication status')
+        .perPage(10)
+        .fields([
+            nga.field('name'),
+            nga.field('published', 'boolean').label('Is published ?')
+        ])
+        .listActions(['show'])
+        .order(4)
+    )
+);
+```
+
+You can add any number of collections you want, even several collections per entity (e.g. to show most commented posts and posts waiting for publication).
+
+As soon as you define a custom dashboard configuration, it will replace the default dashboard based on all entities and listViews.
+
+If you just need to remove one panel from the default dashboard layout, you have to write a custom dashboard configuration from the ground up.
+
+## Dashboard Template
+
+The default dashboard template iterates over all the collections, and displays them from left to right and from top to bottom. If you want to change the size of a particular dashboard panel, or if you want to add panels other than a datagrid, you have to write the dashboard template and set it in the dashboard:
+
+```js
+admin.dashboard(nga.dashboard()
+    .template(templateString)
+);
+```
+
+Here is a copy of the default dashboard template, which you can use as a starting point for a custom dashboard template:
+
+```html
```

Comment:
Can't we just add a link to the dashboard template? I'm affraid this code would be desynchronized over time.